packages/client/wallets/aa/src/blockchain/wallets/smartAccount.ts

```diff
@@ -34,21 +34,34 @@ export function toCrossmintSmartAccountClient<Client extends SmartAccountClient<
             });
         },
         async sendTransaction(txn) {
-            txn = {
-                ...txn,
-                ...(usesGelatoBundler(crossmintChain) && gelatoBundlerProperties),
-            };
-            logInfo(`[CrossmintSmartWallet.sendTransaction] - params: ${JSON.stringify(txn)}`);
-            return smartAccountClient.sendTransaction(txn);
+            return logPerformance("CrossmintSmartWallet.sendTransaction", async () => {
+                try {
+                    txn = {
+                        ...txn,
+                        ...(usesGelatoBundler(crossmintChain) && gelatoBundlerProperties),
+                    };
+
+                    logInfo(`[CrossmintSmartWallet.sendTransaction] - params: ${JSON.stringify(txn)}`);
+                    return await smartAccountClient.sendTransaction(txn);
+                } catch (error) {
+                    throw new TransactionError(`Error sending transaction: ${error}`);
+                }
+            });
         },
         async sendUserOperation({ userOperation, middleware, account }) {
-            userOperation = {
-                ...userOperation,
-                ...(usesGelatoBundler(crossmintChain) && gelatoBundlerProperties),
-            };
+            return logPerformance("CrossmintSmartWallet.sendUserOperation", async () => {
+                const params = {
+                    userOperation: {
+                        ...userOperation,
+                        ...(usesGelatoBundler(crossmintChain) && gelatoBundlerProperties),
+                    },
+                    middleware,
+                    account,
+                };
 
-            logInfo(`[CrossmintSmartWallet.sendUserOperation] - params: ${{ userOperation, middleware, account }}`);
-            return smartAccountClient.sendUserOperation({ userOperation, middleware, account });
+                logInfo(`[CrossmintSmartWallet.sendUserOperation] - params: ${params}`);
```

Comment:
do we have the annotation for `@LogInputOutput` from matias?

Reply:
wdym by annotation here?

Using the `logInputOutput` function rn looks like:
```TS
export const getViemNetwork = logInputOutput((cmChain: EVMBlockchainIncludingTestnet) => {
// function contents here.
}, "getViemNetwork");
```

Reply:
I'm going to merge this one @afeight can refine later if needed!